src/shared/components/results/__tests__/ResultsFacets.spec.tsx

```diff
@@ -43,7 +48,7 @@ describe('ResultsFacets', () => {
     });
   });
 
-  it('should deselect a facet', async () => {
+  test('should deselect a facet', async () => {
```

Comment:
Looking at this change I realised there were 221 occurrences of ` it('` and 221 of ` test('` so we don't really have a consistent approach in naming. I guess it's fine to call them 'it' when the name of the test makes sense ('...should') but I don't really care for one or the other.
Just commenting here, not asking to revert